src/histolab/tiler.py

```diff
@@ -48,6 +48,8 @@
 
 logger = logging.getLogger("tiler")
 
+COORDS_WITHIN_EXTRACTION_MASK_THRESH = 0.8
```

Comment:
mmmm i don't like this name honestly ... it seems too long and truncated ... can we find something more clever?

Reply:
ehm this is exactly what it is 🤔  help

Reply:
Ok at this point then i would prefer `COORDS_WITHIN_EXTRACTION_MASK_THRESHOLD` ... it's just > 3 chars 😄